flink-runtime/src/test/java/org/apache/flink/runtime/rest/FileUploadHandlerTest.java

```diff
@@ -50,6 +55,24 @@
 
 	private static final ObjectMapper OBJECT_MAPPER = RestMapperUtils.getStrictObjectMapper();
 
+	private static ResourceLeakDetectorFactory previousLeakDetector;
+	private static ResourceLeakDetector.Level previousLeakDetectorLevel;
+
+	@BeforeClass
+	public static void setLeakDetector() {
```

Comment:
we could move this into a utility `LeakDetectorResource` so we can re-use it for any test by simply adding it as a `@Rule`.

Reply:
great idea - done